codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectAggregator.java

```diff
@@ -268,12 +257,11 @@ public void operationComplete(ChannelFuture future) throws Exception {
 
     @Override
     public void channelInactive(ChannelHandlerContext ctx) throws Exception {
-        super.channelInactive(ctx);
-
-        // release current message if it is not null as it may be a left-over
-        if (currentMessage != null) {
-            currentMessage.release();
-            currentMessage = null;
+        try {
+            super.channelInactive(ctx);
```

Comment:
This rework doesn't make too much sense because this invocation will fall to
```
    private void AbstractChannelHandlerContext#invokeChannelInactive() {
        if (invokeHandler()) {
            try {
                ((ChannelInboundHandler) handler()).channelInactive(this);
            } catch (Throwable t) {
                notifyHandlerException(t);
            }
        } else {
            fireChannelInactive();
        }
    }
```
So, looks like the current call ```invokeChannelInactive``` here is safe in terms of throwing exceptions.